docs/en/development/io_processing.md

```diff
@@ -3,7 +3,51 @@ sidebar_label: Read/Write/Delete Processing Flow
+
+### Writeback mode
+
+Enabling `--writeback` when mounting is also an option to further improve the performance if the data consistency and reliability are not critical. If the writeback mode is enabled, slice flush operation will return immediately after writing data to the local staging directory shared with cache. The data will then be uploaded asynchronously to the object storage by a backend thread. It's worth to be mentioned that, different from the so called _write to memory first_ mechanism, the writeback mode of JuiceFS also writes data into local cache directory (vary from the hardwares and the local file systems the cache directory is on in practice). In other words, the local directory is a cache layer of the object storage.
+
+With the writeback mode enabled, JuiceFS will aggressively preserve all of the data into the cache directory without checking the size of the objects to be uploaded by default. This is useful in some scenarios which generate a huge plenty of intermediate files such as compiling software. In addition, a new parameter `--upload-delay` has been introduced since JuiceFS v0.17, which is for delaying the upload action to locally cache the data in a more aggressive way. If a file has been deleted before uploading, the upload action will ignore this file. This is not only a performance improvement, but also a cost saving. Meanwhile, comparing with local disk, JuiceFS uploads files automatically when the cache directory is running out of space, which keeps the applications away from unexcepted failures. This is useful for the applications requiring temporary file storages such as Spark shuffle.
```

Comment:
```suggestion
With the writeback mode enabled, JuiceFS will aggressively preserve all of the data into the cache directory without checking the size of the objects to be uploaded by default. This is useful in some scenarios which generate a large number of intermediate files such as compiling software. In addition, a new parameter `--upload-delay` has been introduced since JuiceFS v0.17, which is for delaying the upload action and caching the data locally in a more aggressive way. If a file has been deleted before being uploaded, the upload action will ignore this file. This can not only improve performance but also save cost. Meanwhile, compared with a local disk, JuiceFS uploads files automatically when the cache directory is running out of space, which keeps the applications away from unexcepted failures. This is useful for applications requiring temporary file storage such as Spark shuffle.
```